src/main/java/games/strategy/engine/framework/startup/ui/PlayerSelectorRow.java

```diff
@@ -23,27 +23,31 @@
 
 class PlayerSelectorRow {
 
+  private static final String PLAYER_TYPE_AI = "AI";
+  private static final String PLAYER_TYPE_DOES_NOTHING = "DoesNothing";
```

Comment:
Just thinking out loud...  Is there a better place for these constants (including one for `"Human"`)?  Since you're defining the legal values in the DTD, that would suggest they're API and should be defined in the engine domain model.  I'm guessing this feature is still evolving, and you'll probably move them when everything becomes better focused. :smile:  (An `enum` may even be in its future given that the DTD limits it to a finite set of values.)